### PR TITLE
feat: units on header jobs result and selected project header on home…

### PR DESCRIPTION
**Pull request by AlbaLopez**
… menu

### CODE REVIEW

**Comment by AlbaLopez** [comment]
screen shot 2019-01-23 at 11 14 34

File: src/app/jobs/components/job-detail/components/job-result-table/job-results-table.component.html [hunk 1]
```diff
@@ -69,7 +64,7 @@ <h3 class="inline">Results</h3>
 
     <ng-container matColumnDef="product">
       <th mat-header-cell *matHeaderCellDef>
-        <span mat-sort-header>Product</span>
+        <p mat-sort-header>Production <span class="units">[mmmol gDW<sup>-1</sup> h<sup>-1</sup>]</span></p>
```
Comment (kvikshaug) [inline src/app/jobs/components/job-detail/components/job-result-table/job-results-table.component.html]: Typo, `mmmol` should be `mmol`

File: src/app/jobs/components/job-detail/components/job-result-table/job-results-table.component.ts [hunk 2]
```diff
@@ -386,12 +362,53 @@ export class JobResultTableComponent implements AfterViewInit, OnInit, OnDestroy
...
       this.tableData[0].heterologous_reactions.length), this.tableData.reduce((max, row) => row.heterologous_reactions.length > max ?
       row.heterologous_reactions.length : max, this.tableData[0].heterologous_reactions.length)];
-    this.reactionsFilter.patchValue(this.filterValues.reactions);
+    this.reactionsFilter.setValue(this.filterValues.reactions);
 
     this.filterValues.manipulations = [this.tableData.reduce((min, row) => row.manipulations.length < min ? row.manipulations.length : min,
       this.tableData[0].manipulations.length), this.tableData.reduce((max, row) => row.manipulations.length > max ?
       row.manipulations.length : max, this.tableData[0].manipulations.length)];
     this.manipulationsFilter.patchValue(this.filterValues.manipulations);
+    console.log('FILTER VALUES', this.filterValues);
```
Comment (kvikshaug) [inline src/app/jobs/components/job-detail/components/job-result-table/job-results-table.component.ts]: Please remove this

File: src/app/jobs/components/job-detail/components/job-result-table/job-results-table.component.html [hunk 3]
```diff
@@ -131,21 +131,43 @@ <h3 class="inline">Results</h3>
...
+            </a>
+          </div>
+        </div>
+        <div>
+           <a (click)="showAllManipulations = true;" [hidden]="showAllManipulations" style="cursor: pointer; padding-top: 10px">
+          ...
           </a>
         </div>
+        <div style="padding-top: 10px; padding-bottom: 10px;">
+        <span style="color: rgba(0, 0, 0, 0.54); font-size: 12px;">↑ up-regulation, ↓ down-regulation<mat-icon
```
Comment (kvikshaug) [inline src/app/jobs/components/job-detail/components/job-result-table/job-results-table.component.html]: Why do the styles go in directly in the html here and not css classes?

**Review by kvikshaug — COMMENTED** [review]
It would be nice to follow the semantic commit conventions a bit more. For example here you're adding the project header feature in the same commit as the job results units, those are pretty different features. It's easier during development to add everything into one commit, but that makes it harder for others to review and to look back in git history later.

Otherwise it looks good, I like how you put the Caffeine label back in the header and show the selected project permanently now.

File: src/app/jobs/components/job-detail/components/job-result-table/job-results-table.component.html (hunk 3 above)
Comment (AlbaLopez) [reply src/app/jobs/components/job-detail/components/job-result-table/job-results-table.component.html]: Because I am an idiot and I forgot it hahaha

**Comment by kvikshaug** [comment]
The project label should probably be white here like the header:

image

**Comment by AlbaLopez** [comment]
I am checking how to change the color as the same color that has the toolbar :)

**Comment by AlbaLopez** [comment]
screen shot 2019-01-24 at 08 46 02 1
screen shot 2019-01-24 at 08 45 54

**Review by kvikshaug — APPROVED** [review]

**Pull request merged**